src/OrchardCore.Modules/OrchardCore.Users/Drivers/UserDisplayDriver.cs

```diff
@@ -148,6 +149,11 @@ public override async Task<IDisplayResult> UpdateAsync(User user, UpdateEditorCo
                     var token = await _userManager.GenerateEmailConfirmationTokenAsync(user);
                     await _userManager.ConfirmEmailAsync(user, token);
                 }
+                if (model.PhoneNumberConfirmed && !await _userManager.IsPhoneNumberConfirmedAsync(user))
```

Comment:
Again I would remove the confirmation process from here.

Reply:
Replied as above